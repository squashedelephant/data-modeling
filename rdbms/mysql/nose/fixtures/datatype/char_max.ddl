CREATE TABLE data(phrase CHAR(255));
