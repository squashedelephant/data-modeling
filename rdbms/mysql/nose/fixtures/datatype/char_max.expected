We the people, in order to form a more perfect union establish justice ensure domestic tranquility, provide for the common defense, promote the general welfare and ensure the blessings of liberty for ourselves and our prosperity to ordain and establish t
